<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 14px &quot;Microsoft YaHei&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius:30px;
	background-color: white;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
       <item>
        <widget class="QFrame" name="leftFrame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>150</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>150</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>350</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>350</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/img/welcome.svg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="rightFrame">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border: none;
	background: none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="img.qrc">
                <normaloff>:/img/close.svg</normaloff>:/img/close.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="signinPage">
             <widget class="QFrame" name="loginForm">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>481</width>
                <height>701</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <property name="leftMargin">
                <number>50</number>
               </property>
               <property name="rightMargin">
                <number>50</number>
               </property>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="styleSheet">
                  <string notr="true">QLabel{
	font-size: 32px;
	font-weight: bold;
}</string>
                 </property>
                 <property name="text">
                  <string>Welcome back!</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit{
	font-size: 16px;
	border-radius: 8px;
	background: whitesmoke;
	padding: 10px;
	min-height:40px;
	max-height:40px;
}</string>
                 </property>
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>User</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_2">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit{
	font-size: 16px;
	border-radius: 8px;
	background: whitesmoke;
	padding: 10px;
	min-height:40px;
	max-height:40px;
}</string>
                 </property>
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QCheckBox" name="checkBox">
                   <property name="text">
                    <string>Remember me</string>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="styleSheet">
                    <string notr="true">QPushButton{
	font-weight: bold;
	border: none;
	background: none;
	text-decoration: underline;
}</string>
                   </property>
                   <property name="text">
                    <string>Forgot my password</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="signinButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	font-size: 18px;
	font-weight: bold;
	border-radius: 30px;
	background: #536DFE;
	padding: 10px;
	min-height:40px;
	max-height:40px;
}</string>
                 </property>
                 <property name="text">
                  <string>Sign in</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Don't have an account?</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="goSignupButton">
                   <property name="styleSheet">
                    <string notr="true">QPushButton{
	font-weight: bold;
	border: none;
	background: none;
	text-decoration: underline;
}</string>
                   </property>
                   <property name="text">
                    <string>Sign up</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="signupPage">
             <widget class="QFrame" name="loginForm_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>481</width>
                <height>701</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <property name="leftMargin">
                <number>50</number>
               </property>
               <property name="rightMargin">
                <number>50</number>
               </property>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">QLabel{
	font-size: 32px;
	font-weight: bold;
}</string>
                 </property>
                 <property name="text">
                  <string>Create Your Account</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_3">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit{
	font-size: 16px;
	border-radius: 8px;
	background: whitesmoke;
	padding: 10px;
	min-height:40px;
	max-height:40px;
}</string>
                 </property>
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>User</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_4">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit{
	font-size: 16px;
	border-radius: 8px;
	background: whitesmoke;
	padding: 10px;
	min-height:40px;
	max-height:40px;
}</string>
                 </property>
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>44</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="signupButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	font-size: 18px;
	font-weight: bold;
	border-radius: 30px;
	background: #536DFE;
	padding: 10px;
	min-height:40px;
	max-height:40px;
}</string>
                 </property>
                 <property name="text">
                  <string>Sign in</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Already have an account?</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="goSigninButton">
                   <property name="styleSheet">
                    <string notr="true">QPushButton{
	font-weight: bold;
	border: none;
	background: none;
	text-decoration: underline;
}</string>
                   </property>
                   <property name="text">
                    <string>Sign in</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
